.github/workflows/release.yml

```diff
@@ -99,9 +99,15 @@ jobs:
       - name: Create release
         run: |
           gh release create ${{ github.ref }} --generate-notes --prerelease
-          for i in release/*/*; do
-            gh release upload ${RELEASE_NAME} $i
-          done
+          # skip upload if there are no files
+          if stat release/*/* >/dev/null 2>&1; then
+            for i in release/*/*; do
+              gh release upload ${RELEASE_NAME} $i
```

Comment:
Should have quotes here around the variables:

    gh release upload "${RELEASE_NAME}" "$i"

Reply:
It would be ideal factor this out as a script and have [shellcheck](https://www.shellcheck.net/) run on all our scripts.